furusiya/model/skills/omnislash.py

```diff
@@ -3,7 +3,19 @@
 
 class OmniSlash:
     @staticmethod
-    def process(player, rehit_percent, delta_coords):
-        should_re_hit = Game.random.randint(0, 100) <= rehit_percent
-        if should_re_hit:
-            player.move_or_attack(*delta_coords)
+    def process(player, target, conf):
+        # do guaranteed hits
+        for _ in range(conf.guaranteedHits):
+            if Game.fighter_system.has(target):
+                Game.fighter_system.get(player).attack(target)
+            else:
+                return  # it's dead already!
```

Comment:
This could be a `break`, no? We're done if it's dead.

Reply:
Yep. In this case, `return` works just as well. /shrug